app/views/passengers/show.haml

```diff
@@ -18,7 +18,7 @@
 - if @passenger.subscribed_to_sms?
   %p
     %b SMS address:
-    = @passenger.phone.delete('-') + @passenger.carrier.gateway_address
+    = @passenger.phone.tr('^0-9', '') + @passenger.carrier.gateway_address
```

Comment:
til the `tr` method. huh.

Reply:
out of curiosity, why not `gsub`?

Reply:
OK never mind I looked it up, `tr` is faster but I didn't think it took regular expressions as an argument. Seems to work though.

Reply:
What was I thinking.